packages/bpk-component-breakpoint/README.md

```diff
@@ -24,17 +24,21 @@ export default () => (
 
 ### Server Side Render (SSR) Support
 
-You can pass `matchSSR` which will instruct the breakpoint to match any time it is rendered on the server.
+You can pass `matchSSR` which will instruct the breakpoint to match any time it is rendered on the server. You can use
+this to estimate what breakpoint is likely to match on the client-side.
 
 ```js
-<BpkBreakpoint query={BREAKPOINTS.TABLET} matchSSR>
+<BpkBreakpoint query={BREAKPOINTS.TABLET} matchSSR={isTablet}>
```

Comment:
I really like this, I didn't know we have this info from the request-context. Maybe we can add an import as well to clarify where the `isTablet` boolean would come from? Maybe it's not just the request-context and can be something else, but just to clarify that it's not part of the API of `BpkBreakpoint`

Reply:
Did think about that, but request-context is something specific to Skyscanner and not open-source, do we lean towards providing Skyscanner-specific docs in backpack?

Reply:
> do we lean towards providing Skyscanner-specific docs in backpack?

No, we don't. I think it's fine to just add any dummy import just to clarify it's not part of the API 🤔

Reply:
added dummy import